types/phoenix/index.d.ts

```diff
@@ -39,15 +39,17 @@ export class Channel {
 export type ConnectionState = 'connecting' | 'open' | 'closing' | 'closed';
 
 export interface SocketConnectOption {
+  binaryType: string;
```

Comment:
do we know all the possible binary types? If so, we may want to list them here.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/binaryType lists "arraybuffer" and "blob", but I'm not sure if that's an exhaustive list. I couldn't find its use in the (elixir) phoenix source code either.

Reply:
MDN tends to have comprehensive docs. The two are probably all the possibilities. Would you like to update the PR? I'll take a look at the Phoenix source tomorrow, too. Past midnight in Australia.

Reply:
I updated the commit to define the `BinaryType` type.